frontend/src/theme.ts

```diff
@@ -234,3 +239,21 @@ export const darkTheme = createTheme({
     },
   },
 })
+
+export const ThemeNames = {
```

Comment:
Generally the naming convention for enums is to use singular names, not plural. The reason being is that when you use the enum in code, you're selecting one thing from the choices available.

I think the same rules should apply here as it's our own way of using enums without all of the TS headache.